org.mixedrealitytoolkit.input/Tests/Editor/PackageValidationTest.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clause
+
+// Disable "missing XML comment" warning for tests. While nice to have, this documentation is not required.
```

Comment:
nit: I personally think inserting some standard, consistent, simple XML headers for the tests looks cleaner than disabling a warning for each file.

Reply:
I can do this for you @keveleigh  :)  For the new stuff at least.